### DISCUSSION TITLE
Fluent session randomly stops when running in terminal?

### DISCUSSION THREAD

**Discussion by cons013** [ansys/pyfluent#3534] [Q&A]
Hello,

I'm running a fluent session in a pycharm terminal with python, and randomly the process just stops:

![image](https://github.com/user-attachments/assets/27de9919-4034-48c7-a565-aa1d190bf6b8)

Why exactly does this happen? I switched away from my screen for <30 seconds and this has happens - I've encountered it a couple of times now. There is no error message at all. I'm not sure what other files to check for debugging.

Best,
cons013

**Comment by mkundu1** [comment]
@cons013 Are able to reproduce it outside PyCharm, e.g., running from a command prompt?

**Comment by ansjschuetz** [comment]
Looks to me as if some Linux process receives a SIGSTOP signal. I'd doubt that this is caused by Ansys software itself...

**Comment by cons013** [comment]
@mkundu1 I left a session running overnight idle and it's still up, will try some different things to replicate it but so far it seems to be okay. May have been a temporary hardware issue?
